### PR TITLE
hard coded phrases, missing notifications when using qb-target, and spanish corrections

### PR DESCRIPTION
**Pull request by mockdot**
Describe Pull request
Removed some missing hard coded phrases, added missing notifications when using qb-target (otherwise you don't understand how to do the job), fixed spelling in spanish language.

Questions (please complete the following information):

Have you personally loaded this code into an updated qbcore project and checked all it's functionality? Yes
Does your code fit the style guidelines? Yes
Does your PR fit the contribution guidelines? Yes

### CODE REVIEW

File: locales/en.lua [hunk 1]
```diff
@@ -23,7 +23,7 @@ local Translations = {
         start_work = 'Start Working',
         stop_working = '[E] Stop Working',
         stop_work = 'Stop Working',
-        grab_stall = '[~g~G~s~] Grab Stall',
+        grab_stall = '[G] Grab Stall',
         drop_stall = '[~g~G~s~] Release Stall',
```
Comment (GodyFromDiscord) [inline locales/en.lua:27]: I am getting [~g~G~s~] when picking the stall up, please change this to just [G]

**Review by GodyFromDiscord — CHANGES_REQUESTED** [review]
I've requested some changes, but the following line should also not be plural as well

currently
made_hotdog = 'You made a %{value} Hot Dogs',

suits the following better
made_hotdog = 'You made a %{value} Hot Dog',

File: locales/en.lua (hunk 1 above)
Comment (mockdot) [reply locales/en.lua:27]: Hmmm no, sorry, that was already there and it also changes the color for when you use DrawText3D. I just fixed the [G] for when you use qb-target. If you are a QBCore developer please decide the design path so you can make an accurate fix to that vs using DrawText3D. This contribution is to add missing key information for when using qb-target.

Comment (GodyFromDiscord) [reply locales/en.lua:27]: I will take a further look into this, thank you.

**Pull request closed without merging**